lib/v2/gogoanimehd/recent-releases.js

```diff
@@ -0,0 +1,38 @@
+const got = require('@/utils/got');
+const cheerio = require('cheerio');
+
+module.exports = async (ctx) => {
+    const rootUrl = 'https://gogoanimehd.io';
+
+    const response = await got({
+        method: 'get',
+        url: rootUrl,
+    });
+
+    const $ = cheerio.load(response.data);
+    const recentReleases = $('.last_episodes');
+    const listItems = $(recentReleases).find('li');
+
+    const arrayOfItems = listItems.toArray().map((item) => {
+        const title = $(item).find('.name a').attr('title');
+        const episode = $(item).find('.episode').text();
+        const link = $(item).find('.name a').attr('href');
+        const img = $(item).find('.img a img').attr('src');
+
+        const formattedTitle = `<div>${title}<br/><h2>${episode}</h2></div>`;
```

Comment:
Title only accepts a plain string.

Reply:
oh :(
can i add the html and the episode value in the description ?